README.md

```diff
@@ -19,10 +19,11 @@ It records a spoken request from the connected microphones, sends it to the Goog
 ## Run the sample
 
 - Configure the Google API Console Project to use the [Google Assistant API][google-assistant-api-config].
-- Download the `client_secret_NNNN.json` file from the [credentials section of the Console][console-credentials].
+- Download the `client_secret_NNNN.json` file from the [credentials section of the Console][console-credentials]. Make sure to use "other" option and not "Android" when creating Oauth Client
+- Install `click` Python package, `pip install click` (required for the google-oauthlib-tool)
```

Comment:
Checking `pip freeze`, I do appear to have the package click. But I'd rather not have this instruction here and fix the dependency requirements in the tool itself. I'll check the setup for that.

Reply:
Looks like package is already required when installing [the tool](https://github.com/GoogleCloudPlatform/google-auth-library-python-oauthlib/blob/master/setup.py#L21)

Reply:
Didn't install `click` automatically for me

Reply:
I don't know why it didn't install, but I wouldn't want to add in an additional step that isn't necessary for most users.

Reply:
@championswimmer it should install it if you installed it with `pip install google-auth-oauthlib[tool]`